### DISCUSSION TITLE
Mirth is running but unable to login to MirthConnect Administrator

### DISCUSSION THREAD

**Discussion by KarolLipnicki** [nextgenhealthcare/connect#6129] [General]
Hi

Our production environment runs two mirth servers (on separate servers) connected to one Postgres database. From time to time, we observe a situation where both of them work properly (process events in queues), but one of them cannot be logged in, and when trying to access Mirth Connect Administrator, the process hangs on the "logging" message. The logs do not show any related errors at this time

OS: Red Hat Enterprise Linux release 9.3 (Plow)
Mirth version: 4.4.2.b326-1
Java version: openjdk 21.0.2 2024-01-16 LTS

**Comment by pacmano1** [comment]
Without the clustering plugin, your setup is not officially supported.   I do believe people do run a setup like yours, perhaps others will chime in.   Perhaps it is a server id problem.
